buddyboss/buddyboss-platform
# Member page become slow for thousand of users.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1202 .

### How to test the changes in this Pull Request:

1. Go To members page
2. See error https://prnt.sc/ti69if

### Proof Screenshots or Video

https://prnt.sc/10ly1ho

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
